CHANGELOG.md

```diff
@@ -1,5 +1,27 @@
 # Changelog
 
+## 0.7.3  (September 20, 2019)
+
+### Added
```

Comment:
What are your thoughts on rearranging to Added, Removed, Fixed? I just assume Removed is more important to users than bug fixes.

Otherwise this looks good to me.

Reply:
Sure! I’ll update the order. :)